src/segment_set.rs

```diff
@@ -48,20 +48,21 @@
 //! would make changing the beginning and end at the same time faster, and is
 //! attractive future work.
 
-use database::DbOptions;
-use database::Executor;
-use database::Promise;
-use diag::Diagnostic;
+use crate::database::DbOptions;
+use crate::database::Executor;
+use crate::database::Promise;
+use crate::diag::Diagnostic;
```

Comment:
What's the reason for all these `crate::` prefixes? Is your editor making you do that or something? It should work without it.

Reply:
I think this came from switching to Rust 2018 Edition. There is a remark [here](https://doc.rust-lang.org/reference/paths.html#path-qualifiers).
You're right that it shall not be required, however, so I'll try to revert them.

Reply:
Ok, it's official, the Rust 2018 Edition requires those `crate::`, so they have to be kept for Rust 2018.
```
error[E0432]: unresolved import `diag`
  --> src/verify.rs:28:5
   |
28 | use diag::Diagnostic;
   |     ^^^^ help: a similar path exists: `crate::diag`
   |
   = note: `use` statements changed in Rust 2018; read more at <https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html>

error: aborting due to previous error
```
So, the discussion might be whether we want Rust 2015 or Rust 2018. [I'm for Rust 2018.](https://github.com/david-a-wheeler/metamath-knife/pull/9/commits/29f2d8a2bcfe36fbd8ba32a16e78a98de2b6c508)

Reply:
Oh I see. The `crate` is not required, but they are relative paths without it, so accessing a sibling module requires either `crate` or `super`. I'm fine with using `crate` in this case.